jwql/tests/test_plotting.py

```diff
@@ -36,3 +46,28 @@ def test_bar_chart():
     plt = bar_chart(data, 'index')
 
     assert str(type(plt)) == "<class 'bokeh.plotting.figure.Figure'>"
+
+
+@pytest.mark.skipif(sys.version_info[:2] != (3, 6),
+                    reason="Web server run on Python 3.6")
+def test_bokeh_version():
+    """Make sure that the current version of Bokeh matches the version being
+    used in all the web app HTML templates.
+    """
+    env_version = bokeh.__version__
+
+    template_paths = os.path.join(JWQL_DIR, 'website/apps/jwql/templates', '*.html')
+    all_web_html_files = glob.glob(template_paths)
+
+    for file in all_web_html_files:
+        with open(file) as f:
+            content = f.read()
+
+        # Find all of the times "bokeh-#.#.#' appears in a template
+        html_versions = re.findall(r'(?<=bokeh-)\d+\.\d+\.\d+', content)
```

Comment:
@laurenmarietta Would this also include instances of `bokeh-widgets-#.#.#` as well?  I think we have a few of those scattered around our templates.

Reply:
Good idea!